frappe/geo/country_info.json

```diff
@@ -1019,9 +1040,11 @@
  },
  "Guinea-Bissau": {
   "code": "gw",
+  "currency": "XOF",
+  "currency_name": "West African CFA Franc",
+  "currency_symbol": "FCFA",
```

Comment:
@tundebabzy can you verify

West African Frank symbol is just `CFA` (https://en.wikipedia.org/wiki/West_African_CFA_franc)

Central African Frank is `FCFA` (https://en.wikipedia.org/wiki/Central_African_CFA_franc)

Reply:
It took me some time of research before I chose `FCFA`. If you check [https://en.wikipedia.org/wiki/Currency_symbol] (this link on wikipedia), it says `FCFA`. I finally settled for FCFA since that's what is written on their coins.

![220px-bceaofranc](https://user-images.githubusercontent.com/818803/30431208-c7bbb6f2-9955-11e7-9b6e-96ae249f5fb4.png)

![220px-africacoins](https://user-images.githubusercontent.com/818803/30431264-f39a4874-9955-11e7-8c3a-cc67f82deca4.png)

By Hayko - Own work, scan, GFDL, https://commons.wikimedia.org/w/index.php?curid=4260402

![screenshot from 2017-09-14 14-07-42](https://user-images.githubusercontent.com/818803/30431427-7188abea-9956-11e7-8f17-64d23ca75e94.png)

Reply:
I would go with Wikipedia. I am okay with the Central African CFA Frank being `FCFA`, the issue is in the West African CFA Frank... I think the screenshot you show also shows `CFA`  for West African?

Reply:
Yes, I must didn't look closely at first. I have changed it to CFA